content/c-sharp/concepts/math-functions/terms/max/max.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'Math.Max()'
+Description: 'Returns the larger of two specified numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
```

Comment:
```suggestion
  - 'Methods'
  - 'Numbers'
```

Reply:
I've added the 'Numbers' tag and left 'Arithmetic' tag to stay because i thought the more the better.

Reply:
Is there something else i should fix? I don't know what to do.